src/components/home/HeroSection.vue

```diff
@@ -0,0 +1,79 @@
+<script setup lang="ts">
+import BaseButton from "@/components/base/BaseButton.vue";
+</script>
+
+<template>
+  <div class="hero">
+    <div class="hero__content">
+      <h1 class="heading-1">Welcome to the best cinema in Monterail</h1>
+      <p class="paragraph">
+        Best movies, best experiences, best people. And all that doesn’t exist.
+        Yikes.
+      </p>
+      <BaseButton to="#" button-size="large">Book a ticket</BaseButton>
+    </div>
+    <div class="hero__image">
+      <img src="@/assets/images/welcome.jpg" alt="feature image" />
```

Comment:
When adding `alt` try adding more descriptive text that could help people understand what image is showing

https://help.siteimprove.com/support/solutions/articles/80000863904-accessibility-image-alt-text-best-practices

Reply:
Thank you. I will definitely read up on this - not my strongest point